Thorium-Sim/thorium-nova
# Star map editor

## Description

Starts work on the star map editor. Includes the interstellar map, the ability to drag map objects around, and the ability to create and destroy objects.

The map can be accessed by navigating to "Plugin Config -> Thorium Default -> Edit -> Universe"

## Related Issue








Closes #181
Closes #180
Closes #187
Refs #179 
Refs #172


## How do you know the changes work correctly?






Added a simple automated test.
Manual testing:
- Navigate to the universe editor
- Click the "Add" button in the menubar
- Drag the solar system around
- Refresh the browser - the solar system should be in the same place it was dragged to
- Delete the solar system
-

## Screenshots (if appropriate):

## Types of changes



      change)

## Checklist:
